src/Resources/config/services.xml

```diff
@@ -32,5 +35,22 @@
                 </service>
             </argument>
         </service>
+
+        <service id="webpack_encore.entrypoint_lookup.warmer" class="Symfony\WebpackEncoreBundle\CacheWarmer\EntrypointCacheWarmer">
+            <tag name="kernel.cache_warmer" />
+            <argument />
+            <argument type="service" id="Symfony\Component\DependencyInjection\ContainerInterface" />
+        </service>
+
+        <service id="webpack_encore.cache_fallback" class="Symfony\Component\Cache\Adapter\FilesystemAdapter">
+            <argument />
+            <argument>0</argument>
+            <argument key="$directory">%kernel.cache_dir%/encore/</argument>
+        </service>
+
+        <service id="webpack_encore.cache" class="Symfony\Component\Cache\Adapter\PhpArrayAdapter">
```

Comment:
I think we’ll need a new config option to opt into this behavior. If that config option is false, I think we calcium’s just remove this service in the Extension class. Then we just need the on-invalid behavior on the setCache calls to be correct so Symfony just skips the call instead of an error

Reply:
Do you know what you want the config option to be named?

Reply:
Probably could be cache: true on the “root” level of the bundle config. IMO, no need to make it possible to disable/enable cache independently for each build - someone can ask for that later. Just one global option.